### DISCUSSION TITLE
Requesting a cool way to show "Languages Used" on the Profile page similar to a Repo

### DISCUSSION THREAD

**Discussion by bright-cloud-studio** [community/community#66998] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I find the Profile page to be pretty bare bones. I would like some cooler self-reflecting sections, one specifically would be a "Languages" section similar to what you see on a Repo's page. It would be cool to have some sort of chart or graph that shows, overall, what kind of programming languages you use and the percentages. Seems kind of surprising something like this doesn't exist already, considering it is on Repo pages.

**Comment by seyLu** [comment, marked as answer]
Hi @bright-cloud-studio,

There is already an existing proj that does that, you can check out [github-readme-stats](https://github.com/anuraghazra/github-readme-stats).

You can also customize your profile page by adding a README.md on a repo with the same name as your username. You can check out [awesome-github-profile-readme](https://github.com/abhisheknaiidu/awesome-github-profile-readme) for inspiration.

**Reply by bright-cloud-studio** [reply]
That is awesome! Thanks so much for showing me this, I really appreciate it!
